src/ui/autocomplete.ts

```diff
@@ -168,6 +168,9 @@ class AutocompleteUI {
       this.container.appendChild(this.wrapper);
     }
 
+    // disable browser autofill
+    this.inputField.setAttribute('autocomplete', 'off');
+
```

Comment:
do we wanna pass this in as an option to in case someone wants to use autofill?

Reply:
I think in that scenario, they can use their own input field.